dsbridge/DUIwebview.m

```diff
@@ -107,7 +107,7 @@ - (void)webView:(UIWebView *)webView didCreateJavaScriptContext:(JSContext *)ctx
     ctx[@"_dsbridge"]=^(NSString * method,NSString * args){
         return [JSBUtil call:method :args JavascriptInterfaceObject:JavascriptInterfaceObject jscontext:ctx_];
     };
-    [ctx evaluateScript:INIT_SCRIPT];
+    // [ctx evaluateScript:INIT_SCRIPT];
```

Comment:
should we totally drop UIwebview since we only consider iOS 8+?

Reply:
Yeah, lets use WK web view